src/main/resources/application.yml

```diff
@@ -30,6 +32,20 @@ spring:
       okhttp:
         enabled: true
 folio:
+  environment: ${ENV:FOLIO}
```

Comment:
in 2 other places default value was from lower case folio, register can matter, lets use the same value for all places

Reply:
Yeah, I think typing in lower case allows us to run unit tests at the same time when vagrant box is running. switched to lowercase